### ISSUE TITLE
Positioning watermark larger than main image

### ISSUE THREAD

**Issue by atruttmann** [thumbor/thumbor#1560]
### Thumbor request URL
http://localhost:8888/unsafe/1200x800/filters:watermark(http://localhost:8888/unsafe/1440x960/https%3A%2F%2Fgithub.com%2Fthumbor%2Fthumbor%2Fraw%2Fmaster%2Fexample.jpg,0,-120,0)/https%3A%2F%2Fgithub.com%2Fthumbor%2Fthumbor%2Fraw%2Fmaster%2Fexample.jpg

### Expected behaviour
This is an edge case for typical watermark behavior. I want to watermark an image that is larger than the main image/final output size. I'm running into some issues with positioning the watermark. In the diagram below, I want to position the watermark so both its top and bottom are outside of the main frame. Since Thumbor positions positive numbers from the top and negative numbers from the bottom, I can think of no way to do this.

![Frame 214](https://user-images.githubusercontent.com/16278960/234114850-bb127058-fbf8-4155-a41b-30428b0923e6.png)

### Actual behaviour
If I position the watermark with a y value of -120, the image is positioned from the bottom. If I position it with a y value of 50, it is positioned from the top. This is as expected for Thumbors current setup. In an ideal scenario, I would prefer to watermark the image using the top left as (0,0) and position accordingly.

### Operating system
OSx

**Comment by scorphus** [comment]
Hey, it would be great to visualize what you mean. At least for me, it's a bit hard to follow your thought process — my fault, as English is not my mother tongue. Would you kindly provide actual sample image and watermark along with the expected result — as an actual image, one that you might be kindly able to generate using GIMP/Photoshop. Thanks!
